Web/wwwroot/Scripts/Main.js

```diff
@@ -352,10 +354,11 @@ function play(currentBuffer, loopCondition, offset=0) {
 
     if (currentSource !== undefined) {
         if (!isLoaded) {
-            isLoaded = true; 
+            isLoaded = true;
+            console.log('we are here');
             return;
         }
-        currentSource.stop();
+        currentSource.buffer = undefined;
```

Comment:
avoid assignment of undefined. It's for cases when variable really not initialized.
For other cases better to use null
explanation
https://medium.com/@arieldi/anyway-stop-setting-your-variables-to-undefined-difference-undefined-vs-null-25f15d907fb4